Activities/Cryptography/UiPath.Cryptography.Activities/NetCore/ViewModels/KeyedHashFileViewModel.cs

```diff
@@ -89,54 +88,70 @@ protected override void InitializeModel()
             KeySecureString.IsVisible = false;
             KeySecureString.OrderIndex = propertyOrderIndex++;
 
-            KeyInputModeSwitch.IsVisible = false;
+            var useKeyMenuAction = new MenuAction
+            {
+                DisplayName = "Use Key",
```

Comment:
add this as resources for localization - or check if there are exising resources already